### DISCUSSION TITLE
Is there a way to have "warning" errors?

### DISCUSSION THREAD

**Discussion by tttp** [react-hook-form/react-hook-form#2294] [Q&A] [answered]
Hi,
onBlur, I'd like to add some checks that aren't hard errors, for instance if one type bob@gmal.com, writing a "are you sure it's not bob@gmail.com?" ... but let the user put bob@gmal.com if they are really sure it's not a typo

Is there a way to do that already?

**Comment by bluebill1049** [comment, marked as answer]
not, at the moment. you will have to keep that logic in your app.

**Reply by bluebill1049** [reply]
here is feature request you can vote on: https://github.com/react-hook-form/react-hook-form/issues/1761
